protoc-gen-swagger/genswagger/template_test.go

```diff
@@ -4,12 +4,13 @@ import (
 	"reflect"
 	"testing"
 
+	"fmt"
+
```

Comment:
Actually if this is being moved, maybe move it together with `"testing"` and `"reflect"` and others? Seems strange to have it hang on its own

Reply:
Didn't even realise I had changed this, thanks linters!